front/app/components/admin/ActionForm/FlowVisualization/Blocks/SSOConfigModal.tsx

```diff
@@ -57,6 +62,53 @@ const SSOConfigModal = ({
       width={'550px'}
     >
       <Box m="20px">
+        <Box>
+          <Text fontWeight="bold">
+            <FormattedMessage
+              {...messages.howOftenShouldThisActionBeVerified}
+            />
+          </Text>
+          <Radio
```

Comment:
The radio inputs should be in a radio group, I believe.

Reply:
Hmm, we don't have a `RadioGroup` component in the component library though. And I only see it used in a few very specific situations, where we have to import it from the `HookForm` folder. In the vast majority of cases where we use `Radio` we don't use it. Is it important for a11y or something like that?

Reply:
Yes, I think you are right. It's important for using react hook form but in this case, I guess we are not so not relevant 👍 I'm not sure whether this have any accessibility implications but in the admin those are not critical in any case.